test/Microsoft.AspNet.Mvc.FunctionalTests/project.json

```diff
@@ -54,7 +54,8 @@
         "MvcTagHelpersWebSite": "1.0.0",
         "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-*",
         "xunit.runner.kre": "1.0.0-*",
-        "Microsoft.AspNet.WebUtilities": "1.0.0-*"
+        "Microsoft.AspNet.WebUtilities": "1.0.0-*",
+        "LowercaseUrlsWebSite": "1.0.0-*"
```

Comment:
Keep these in alphabetical order (put your new one where it belongs in the order)